java/src/main/java/ru/nk/training/HelperMethods.java

```diff
@@ -0,0 +1,29 @@
+package ru.nk.training;
+
+import java.util.Objects;
+import java.util.Optional;
+
+public class HelperMethods {
+    @SafeVarargs
+    public final <T> T avoidNull(T firstValue, T... otherValues) {
```

Comment:
It is hard to understand what the method does from the name. It may be better to name this method as "firstNotNull", and "firstNotNull" - as "firstNotNullOptional" or something like that